apps/web/components/booking/SlotPicker.tsx

```diff
@@ -133,10 +133,7 @@ export const SlotPicker = ({
     eventTypeId: eventType.id,
     eventTypeSlug: eventType.slug,
     usernameList: users,
-    startTime:
-      browsingDate === undefined || browsingDate.get("month") === dayjs.tz(undefined, timeZone).get("month")
-        ? dayjs.tz(undefined, timeZone).subtract(2, "days").startOf("day")
```

Comment:
This was the cause of the "bookings can't be made in the past" error. For example, if it is March 15th 3 pm, when looking for slots the start date would be March 13th 3 pm. This fails the `isOutOfBounds` check. Why was there a reason we started slots 2 days in the past?

Reply:
@roae may have more context of this

Reply:
I'm really not sure why that is done that way, I kept it because of that, I guess, it subtracts that time for utc- time zones.

Reply:
@zomars with @roae's comment are we good to go?